### PR TITLE
Regenerate all APIs

### PR DESCRIPTION
**Pull request by jskeet**
These now include async request-taking methods which take
a CancellationToken.

Mike, could you check that the BigtableClient changes are okay?

### CODE REVIEW

File: apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClient.cs [hunk 1]
```diff
@@ -290,6 +352,22 @@ public sealed partial class BigtableClientImpl : BigtableClient
...
+        {
+            if (request.AppProfileId == null)
+            {
+                request.AppProfileId = _appProfileId;
+            }
+
+            return GetUnderlyingClient().MutateRowAsync(request, cancellationToken);
+        }
+
+        => MutateRowAsync(
```
Comment (chrisdunelm) [inline apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClient.cs]: OK, not sure what's happened here.
Will look into it.

Comment (evildour) [reply apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClient.cs]: This is from the Bigtable custom generator, not the normal generator. I'll look into this one.

Comment (chrisdunelm) [reply apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClient.cs]: Ah right, thanks. That's a relief!

**Comment by evildour** [comment]
Other than the generation error, the Bigtable changes look fine, but there's some unnecessary overrides being generated in the `BigtableClientImpl` class created by the custom generator. I can fix this in another PR, but this can be submitted before that. They are harmless and the same thing will happen with or without them.

**Comment by jskeet** [comment]
We can't submit this PR as-is, as the build is currently broken... will look at your PR ASAP Mike.

**Comment by jskeet** [comment]
Regenerated with the new BigTable codegen. Will merge on green.

**Pull request merged**